5_data_munge.yml

```diff
@@ -0,0 +1,76 @@
+target_default: 5_data_munge
+
+packages:
+  - scipiper
+  - dplyr
+  
+sources:
+  - 5_data_munge/src/munge_data_files.R
+  
+targets:
+  5_data_munge:
+    depends:
+      - 5_data_munge/out/wqp_daily_depths.rds.ind
+      - 5_data_munge/out/wqp_daily_nodepths.rds.ind
+    
+# -- For WQP and NWISuv, first do a coarse QA, then reduce to daily  -- #
+# -- For NWISdv, do the coarse QA  -- #
+
+  min_value:
+    command: I(0)
+    
+  max_value:
+    command: I(35)
+    
+  max_daily_range:
+    command: I(30)
+    
+  # first, wqp data # 
+  5_data_munge/out/wqp_daily_depths.rds.ind:
+    command: munge_wqp_withdepths(
+      in_ind = '1_wqp_pull/out/wqp_data.rds.ind',
+      min_value, max_value, max_daily_range, 
+      out_ind = target_name)
+   
+  5_data_munge/out/wqp_daily_depths.rds:
+    command: gd_get('5_data_munge/out/wqp_daily_depths.rds.ind')
+    
+  5_data_munge/out/wqp_daily_nodepths.rds.ind:
+    command: munge_wqp_withoutdepths(
+      in_ind = '1_wqp_pull/out/wqp_data.rds.ind',
+      min_value, max_value, max_daily_range, 
+      out_ind = target_name)
```

Comment:
This works as expected, right? I had it in my head that `target_name` didn't work with named arguments unless it was the first argument, due to a remake quirk (see https://github.com/richfitz/remake/issues/173). But maybe it's only a problem when some arguments are named but others aren't, or when you deviate from the default argument order, or something.

Reply:
It built okay, so I'm guessing one of these hypotheses is correct. I run into this occasionally and beat my head against the wall until I remember this quirk...